dev-libs/json_c/json_c-0.13.recipe

```diff
@@ -0,0 +1,74 @@
+SUMMARY="A JSON implementation in C"
+DESCRIPTION="JSON-C implements a reference counting object model that allows \
+you to easily construct JSON objects in C, output them as JSON formatted \
+strings and parse JSON formatted strings back into the C representation of \
+JSON objects."
+HOMEPAGE="https://github.com/json-c/json-c/wiki"
+COPYRIGHT="2009-2012 Eric Haszlakiewicz
+	2004, 2005 Metaparadigm Pte Ltd"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/json-c/json-c/archive/json-c-$portVersion.tar.gz"
+CHECKSUM_SHA256="259e3a81a804246b128fba7117ab6d261314b33e794cc4d2db71a0b59690d18b"
+SOURCE_DIR="json-c-json-c-$portVersion"
+
+ARCHITECTURES="?x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion="4.0.0"
```

Comment:
See [`Makefile.am`](https://github.com/json-c/json-c/blob/2849650f1130f97621e39ff4cb5b6d3da0e97653/Makefile.am#L45)

```diff
-libVersion="4.0.0"
+libVersion="3.0.1"
```

Reply:
Hmm, we are getting a `libjson-c.so.4.0.0` and a `libjson-c.so.4` symlink pointing to it.
So I guess the `version-info 3:1:0` is not what the libjson_c uses for its soname.

Reply:
Because he's not using the correct [release tarballs][1]. The tarball is the [json-0.13][0] branch instead, which would subject to unwanted changes at anytime in the future

[0]: https://github.com/json-c/json-c/tree/json-c-0.13
[1]: https://github.com/haikuports/haikuports/pull/2185/files#r166152061